src/routes/ProfessionScreen.tsx

```diff
@@ -1,62 +1,57 @@
 import React, { ReactElement, useState } from 'react'
 import Header from '../components/Header'
 import MenuItem from '../components/MenuItem'
-import { FlatList, StyleSheet, Text, View } from 'react-native'
+import { FlatList, Text} from 'react-native'
 import axios from '../services/axios'
 import { ENDPOINTS, ProfessionType } from '../constants/endpoints'
 import { SafeAreaInsetsContext } from 'react-native-safe-area-context'
 import { RouteProp, useFocusEffect } from '@react-navigation/native'
 import Loading from '../components/Loading'
 import { COLORS } from '../constants/colors'
-import { widthPercentageToDP as wp } from 'react-native-responsive-screen'
+import { widthPercentageToDP as wp, heightPercentageToDP as hp } from 'react-native-responsive-screen'
 import { RoutesParamsType } from '../navigation/NavigationTypes'
 import { StackNavigationProp } from '@react-navigation/stack'
 import AsyncStorage from '../services/AsyncStorage'
 import labels from '../constants/labels.json'
+import styled from 'styled-components/native'
 
-export const styles = StyleSheet.create({
-  root: {
-    backgroundColor: COLORS.lunesWhite,
-    height: '100%'
-  },
-  text: {
-    marginTop: 56,
-    textAlign: 'center',
-    fontSize: wp('4%'),
-    color: COLORS.lunesGreyMedium,
-    fontFamily: 'SourceSansPro-Regular'
-  },
-  list: {
-    width: '100%'
-  },
-  title: {
-    marginBottom: 32
-  },
-  clickedItemDescription: {
-    fontSize: wp('4%'),
-    fontWeight: 'normal',
-    color: COLORS.white,
-    fontFamily: 'SourceSansPro-Regular'
-  },
-  description: {
-    fontSize: wp('4%'),
-    fontWeight: 'normal',
-    color: COLORS.lunesGreyMedium,
-    fontFamily: 'SourceSansPro-Regular'
-  }
-})
+const Root = styled.View`
+background-color: ${COLORS.lunesWhite};
+height: 100%;
+`;
```

Comment:
formatting

Reply:
Can you explain what you mean by formatting ?

Reply:
This is a hint, that your formatting is wrong. Here the semicolon in line 21 is to much, please remove this.